tests/syscall.rs

```diff
@@ -31,10 +31,18 @@ fn syscall() -> CrabResult<()> {
         )
         .unwrap();
 
+    assert!(target.memory_maps()?.contains(&addr));
```

Comment:
Maybe something like
```suggestion
    assert!(target.memory_maps()?.iter().any(|map| map.address.0 == addr));
```
works?